src/steamship/data/tags/tag_constants.py

```diff
@@ -1,8 +1,21 @@
 from typing import Optional
 
 
+class TagKind:
+    """A set of `kind` constants for Tags."""
+
+    pos = "pos"  # Part of speech tags
```

Comment:
If we want to turn this into an Enum it's recommended to use all caps hooks. 

```suggestion
    POS = "pos"  # Part of speech tags
```

Reply:
Can we spell out all of the tag names and values?  Every NLP human knows POS = Part of speech, but your avg software dev may not.